source/extensions/stat_sinks/dog_statsd/BUILD

```diff
@@ -5,16 +5,17 @@ licenses(["notice"])  # Apache 2
 
 load(
     "//bazel:envoy_build_system.bzl",
-    "envoy_cc_library",
+    "envoy_cc_extension",
     "envoy_package",
 )
 
 envoy_package()
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
     hdrs = ["config.h"],
+    security_posture = "unknown",
```

Comment:
Should we have a thing that is "not applicable" for things outside of the data path? Such as stats or overload managers? Although I suppose stats, tracers, etc. require trusted upstreams.

Reply:
Yeah. I think that some extensions, e.g. access loggers, still do depend on data plane, and so need to be treated the same way as regular data plane extensions like HTTP filters. Others, like stats, tracers, overload managers should not have a data plane dependency. I will tag them as `data_plane_agnostic`. If bugs are discovered in these that make them vulnerable to the data plane, they should be CVE eligible.

Reply:
Actually, tracers are data plane, e.g. the X-Ray tracer makes decisions based on wildcard matching traffic. I will move tracers to a data plane consideration.